lib/vagrant-windows-domain/templates/runner.ps1.erb

```diff
@@ -4,6 +4,24 @@ $credentials = New-Object System.Management.Automation.PSCredential ("<%= option
 <% end %>
 <% if options[:add_to_domain] === true %>
 Add-Computer <%= options[:parameters] %> -Verbose -Force
+<% if options[:rename_trigger] === true %>
+$completed = $false
+while ( -not $completed) {
+	try {
+		Rename-Computer <%= options_rename[:parameters] %> -Verbose -Force -ErrorAction Stop
+		$completed = $true
+	} catch {
+		if ($retrycount -ge 5) {
```

Comment:
I've been out of PoSH game for a while - should this be defined before being used or is this part of the its magic/sillyness?